plans/jenkins/jenkins.yaml

```diff
@@ -120,7 +120,6 @@ jenkins:
         - jenkins-server-security-group
         - install-jenkins-server
         - jenkins-service-account-gcp
-        # - jenkins-service-account-azure
```

Comment:
Curious why this is commented out?  Do we not need it? Oversight?

Reply:
I'll clean this up.  We don't need it; this is probably just old cruft that I missed.  Thx for spotting!